### ISSUE TITLE
firefox+browserpass "is not allowed to refer to the following paths:

### ISSUE THREAD

**Issue by mdorman** [NixOS/nixpkgs#45500]
## Issue description

Trying to install `firefox` results in the error:

```
replacing old 'firefox-61.0.2'
installing 'firefox-61.0.2'
these derivations will be built:
  /nix/store/mjnv9x2vqvmcmnr3jbwf3f8a2109hp0z-firefox-61.0.2.drv
building '/nix/store/mjnv9x2vqvmcmnr3jbwf3f8a2109hp0z-firefox-61.0.2.drv'...
output '/nix/store/z0p59dnrqg3k56l0dk8nihwnpfxc9822-firefox-61.0.2' is not allowed to refer to the following paths:
    /nix/store/9z650yzczx5zyhcw5n36ya6x4jq436mr-gcc-wrapper-7.3.0
```

### Steps to reproduce

In my `~/.config/nixpkgs/config.nix`, I set `firefox.enableBrowserpass = true;`

Then I try to `nix-env --install --attr firefox`.

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.18.4, NixOS, 18.09.git.d89cffb (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.0.4`
 - channels(root): `""`
 - channels(mdorman): `""`
 - nixpkgs: `/var/nixup/nixpkgs`

**Comment by NeQuissimus** [comment]
This is independent from browserpass from what I can see

**Comment by 7c6f434c** [comment]
Yes, and it is probably fixed on staging by #45442

**Comment by vcunat** [comment]
Fixed on master since 7db611f2af.
